scrubble-client/src/styles/pages/game.scss

```diff
@@ -0,0 +1,142 @@
+$base-color: #404040;
+$logo-color: white;
+$button-color: lightgray;
+$head-margins: 1rem;
+
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
```

Comment:
Not important for this PR, but eventually we are going to want to abstract these variables and settings and whatnot into their own files that we can just import into this one

Reply:
sounds good, i may take a stab at this today separate from this PR